### PR TITLE
Adding Migration Doc.

### PR DESCRIPTION
**Pull request by Rajat-0**
Separating Migration doc from README to make it clean and understandable.
closes: #1853

### CODE REVIEW

File: vertical-pod-autoscaler/README.md [hunk 1]
```diff
@@ -54,100 +54,9 @@ as described in the next section.
 Please also have a look there if you want to understand the rationale behind
```
Comment (bskiba) [inline vertical-pod-autoscaler/README.md]: Can you update the previous section too? 
> To receive recommendations please migrate to `autoscaling.k8s.io/v1beta2` API,
as described in the [migration doc]().

File: vertical-pod-autoscaler/README.md [hunk 2]
```diff
@@ -54,100 +54,9 @@ as described in the next section.
...
 the API change.
 
-### Notice on switching to v1beta2 version (0.3.X to >=0.4.0)
+This doc is for installing latest VPA to install previous version or migrate from previous versions
```
Comment (bskiba) [inline vertical-pod-autoscaler/README.md]: Small rewording:
> This doc is for installing latest VPA. For instructions on migration from older versions see [Migration Doc](...)

**Comment by bskiba** [comment]
Thank you! Two minor comments

File: vertical-pod-autoscaler/README.md (hunk 2 above)
Comment (Rajat-0) [reply vertical-pod-autoscaler/README.md]: done

File: vertical-pod-autoscaler/README.md (hunk 1 above)
Comment (Rajat-0) [reply vertical-pod-autoscaler/README.md]: done

File: vertical-pod-autoscaler/README.md [hunk 3]
```diff
@@ -50,104 +50,10 @@ existing recommendations will be removed. The objects will remain present though
 and a ConfigUnsupported condition will be set on them.
 
 To receive recommendations please migrate to `autoscaling.k8s.io/v1beta2` API,
-as described in the next section.
-Please also have a look there if you want to understand the rationale behind
-the API change.
+as described in the [Migration Doc](https://github.com/kubernetes/autoscaler/blob/vpa-release-0.3/vertical-pod-autoscaler/MIGRATE.md)
 
-### Notice on switching to v1beta2 version (0.3.X to >=0.4.0)
+This doc is for installing latest VPA. For instructions on migration from older versions see Migration Doc [Migration Doc](https://github.com/kubernetes/autoscaler/blob/vpa-release-0.3/vertical-pod-autoscaler/MIGRATE.md)
```
Comment (bskiba) [inline vertical-pod-autoscaler/README.md]: nit: You can remove the first Migration Doc (this will only leave the link)

Comment (bskiba) [reply vertical-pod-autoscaler/README.md]: Another thing, let's lint to the master branch, since the vpa-release-0.3 does not have MIGRATE.md

**Pull request merged**